src/AppBundle/Controller/Admin/BlogController.php

```diff
@@ -87,6 +87,11 @@ public function newAction(Request $request)
             $em->persist($post);
             $em->flush();
 
+            // Flash messages are used to notify the user about the result of the
+            // actions. They only last as long as the next request.
```

Comment:
Where can I find the updated explanation? In http://symfony.com/doc/current/book/controller.html#flash-messages I read:

> You can also store small messages that will be stored on the user's session for exactly one additional request. [...]
>
> [...] By design, flash messages are meant to live for exactly one request (they're "gone in a flash").